datahub-web-react/src/app/entity/ownership/ManageOwnership.tsx

```diff
@@ -20,7 +20,9 @@ const PageTitle = styled(Typography.Title)`
     }
 `;
 
-const ListContainer = styled.div``;
+const ListContainer = styled.div`
+    height: calc(100% - 120px);
```

Comment:
Salman - do you mind providing an overview of the changes in the PR description so I can better understand the rationale for solution?

Reply:
I have calculate PageHeaderContainer height with inspecting in browser i.e 120px and trying to remove from ListContainer to make it scrollable. I have check on different resoltion like 1920, 1320 it was properly. but if pixel is concern let me try with vh or Percentage

Reply:
can be replace with height: calc(100% - 11vh); after calculating PageHeaderContainer and Pagination